### ISSUE TITLE
16:10 Resolutions Support & Fixed Player.exe Background

### ISSUE THREAD

**Issue by jdl96** [HumanGamer/WASharpIssues#4]
I use a 16:10 monitor that runs at 1680x1050, and I am unable to find this resolution in the selector. The closest is 1600x900. Would it be possible to add resolutions like these? As far as I know, the only thing this probably affects is the Magic Glove range.

Also, the Player.exe background needs to be changed now that the Vsync option has been added. I've created an updated version of this background that has a taller rectangle if you want to use it: [loader.zip](https://github.com/HumanGamer/WASharpIssues/files/8912289/loader.zip)
I can also do the same edit for the WA1/2/3 Player.exe backgrounds if needed.

**Comment by Aryan807** [comment]
Yes, please create backgrounds for WA1/2/3 players too. And thanks for creating a background for player.exe, it is now added and will be displayed to users from second Beta onwards.

Regarding the 16:10, I'd suspect it won't work as entire code of WAPlayer is currently suitable only for 4:3 or 16:9 resolutions. There are much more things that can be affected e.g the camera range etc. Nevertheless, I'll talk with Alex on this issue and we will see if it could be possible or not.

**Comment by jdl96** [comment]
Backgrounds for WA1/2/3: [WA123Loaders.zip](https://github.com/HumanGamer/WASharpIssues/files/8919438/WA123Loaders.zip)
